### ISSUE TITLE
Make it possible to "Boost" non-verified projects with GIVpower

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#3667]
- The logic (math conditioned by project verfied status) is hard coded in SQL Queries and Procedures.
- E.g. below SQL queries are part of the logic we have pushed to DB to calculate ranks
- We have also some queries in the impact-graph code which are executed per query, they also must be modified.

https://github.com/Giveth/impact-graph/blob/staging/migration/1662915983385-ProjectPowerView.ts
https://github.com/Giveth/impact-graph/blob/staging/migration/1683191367806-CreateProjectInstantPowerView.ts

The above is a summary of points from my chat w/ @aminlatifi 

The Tl;Dr is that it's possible to make this happen, but the changes are critical so we need to go slow & then this will require a lot of testing on staging in order to make sure we don't break anything.

**Comment by MoeNick** [comment]
Is it part of post-MVP QF epic?

**Comment by laurenluz** [comment]
no, this isn't directly related to QF. more GIVeconomy side. I will include it in the GIVeconomy WG proposal

**Comment by MoeNick** [comment]
I put p2 for this, correct me if I was wrong @laurenluz

**Comment by laurenluz** [comment]
hmm well this is blocked by the decisions we make in this forum post: https://forum.giveth.io/t/changing-what-verified-means-on-giveth/1300/5

I would just hold off on prioritizing it until we decide if/when we are doing it at all.

**Comment by jainkrati** [comment]
is this still blocked on the forum post @laurenluz

**Comment by laurenluz** [comment]
this is blocked by us building & implementing the decentralized verification process with attestations....

but if we are able to do preliminary work on it... like creating the two separate tags in the backend "verified" and "givbacks eligible" and then broading our system to handle both... I think we can go for it and even move to dev research!

@jainkrati I think Amin would be a great person to handle it.

**Comment by laurenluz** [comment]
if we need to discuss it more or the requirements are not clear let me know and I can expand upon this issue or we can discuss in the GIVeconomy call.

**Comment by divine-comedian** [comment]
@laurenluz - do we want to redefine this issue as we work on decentralized verification and the new status changes for projects?

**Comment by laurenluz** [comment]
yes definitely, this should be iceboxed until the requirements are clear

**Comment by divine-comedian** [comment]
@laurenluz I think @MohammadPCh has made this possible on the BE and @mohammadranjbarz needs to implement the corresponding givbacks-calculation changes

**Comment by laurenluz** [comment]
would love to test this on staging once it's there!

was there consideration around the design for showing GIVpower "rank" or not? are we showing a "rank" for all projects, and the GIVbacks % for that project is then un-correlated to the GIVpower "rank"? @mohammadranjbarz @divine-comedian

**Comment by mohammadranjbarz** [comment]
@laurenluz  @maryjaf  I thin it's on staging and you can test it now

cc: @divine-comedian

**Comment by divine-comedian** [comment]
@laurenluz 
We didn't make any changes on the GIVpower page, I think it looks good

![image](https://github.com/user-attachments/assets/c646d5bd-7fe5-4d7c-9188-6eff54569cf2)


As you can see though if the project is only Vouched we don't make any reference to GIVbacks % matching amount on the project details page

**Comment by laurenluz** [comment]
I would love to test this, but can you guys link me some projects that are vouched but not GIvbacks eligible? I'm having a hard time finding them to scroll and there is no filter option for that.

**Comment by laurenluz** [comment]
Tested with this project: https://staging.giveth.io/project/testing-the-success-view?tab=givpower

So this is a bit weird... We need to create a separation in the rank on the UI, because projects who are "vouched" but not "GIVbacks eligible"... well their project rank is totally irrelevant... acutally the RANK here is totally irrelevant imo..

For these projects, only the instant boosting matters (not the original, updated every 2 weeks GIVpower rank), we are pretending it does, showing no different w/ a GIVbacks eligible project.

I think this needs a new design in general, but as an MVP for this release, I think we can just remove everything to do with rank (hide/comment out the entire box on the right) if the project is vouched but not GIVbacks eligible.

I can work with Mo on a new design for after we make a final decision on the raffle...
![image](https://github.com/user-attachments/assets/ba8fbf21-3bd3-4292-bcbd-9af94f48d624)

**Comment by mohammadranjbarz** [comment]
> Tested with this project: https://staging.giveth.io/project/testing-the-success-view?tab=givpower
> 
> So this is a bit weird... We need to create a separation in the rank on the UI, because projects who are "vouched" but not "GIVbacks eligible"... well their project rank is totally irrelevant... acutally the RANK here is totally irrelevant imo..
> 
> For these projects, only the instant boosting matters (not the original, updated every 2 weeks GIVpower rank), we are pretending it does, showing no different w/ a GIVbacks eligible project.
> 
> I think this needs a new design in general, but as an MVP for this release, **I think we can just remove everything to do with rank (hide/comment out the entire box on the right) if the project is vouched but not GIVbacks eligible.**
> 
> I can work with Mo on a new design for after we make a final decision on the raffle... ![image](https://private-user-images.githubusercontent.com/75490971/370736308-ba8fbf21-3bd3-4292-bcbd-9af94f48d624.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczMzY2ODEsIm5iZiI6MTcyNzMzNjM4MSwicGF0aCI6Ii83NTQ5MDk3MS8zNzA3MzYzMDgtYmE4ZmJmMjEtM2JkMy00MjkyLWJjYmQtOWFmOTRmNDhkNjI0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI2VDA3Mzk0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhNDYzZGNmNDNmODEyOTAzMzVhOTE2NjUwMWRiZTRiOWExYmQ5MmZkNjY2ZjFlZDg4NWI4NDNkNmQ2YzYwOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.syZOcQuU_zcnz9HqlR2Iqv_IPdijyA2aUA1iTzQOgeI)

@laurenluz 
Got it, Is this ok ?
https://giveth-dapps-v2-81nwc3gcc-givethio.vercel.app/project/testing-the-success-view?tab=givpower

<img width="1512" alt="Screenshot 1403-07-05 at 11 50 32 in the morning" src="https://github.com/user-attachments/assets/0fe23f17-0089-46ce-9ac2-ebd7e5ddb137">

**Comment by maryjaf** [comment]
> **I think we can just remove everything to do with rank (hide/comment out the entire box on the right) if the project is vouched but not GIVbacks eligible.**

https://giveth-dapps-v2-81nwc3gcc-givethio.vercel.app/donate/testing-the-success-view this project is **GIVback eligible** and **not vouched** ,  GIVpower tab isn't shown in project page

**Comment by maryjaf** [comment]
> **I think we can just remove everything to do with rank (hide/comment out the entire box on the right) if the project is **vouched** but not **GIVbacks eligible**.**

I couldn't find a project with this condition , how can I find it ?

**Comment by maryjaf** [comment]
This project is **vouched** and **not givback eligible**, 
https://staging.giveth.io/project/testdraft?tab=givpower
The users can boost to this project but the rank information isn't shown 
Is this the expected result?

![image](https://github.com/user-attachments/assets/9d8df38f-c8fb-4cd8-9766-9b1583776afa)

**Comment by divine-comedian** [comment]
@maryjaf yes!

**Comment by divine-comedian** [comment]
@laurenluz this front-end change is merged to production so I will close this issue - if you have more requirements please re-open this one!

**Comment by laurenluz** [comment]
good with me!
